src/styles/_star.scss

```diff
@@ -0,0 +1,21 @@
+@use './variables';
+
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+
+  &-star {
+    display: inline-block;
+    width: 16px;
+    height: 16px;
+    background: no-repeat url(/src/images/star-active.svg);
```

Comment:
The background image for the stars should be `star.svg` instead of `star-active.svg` for the default state. Ensure the correct image is used for the default star state【20:0†source】.